Makefile.toml

```diff
@@ -69,6 +69,45 @@ end
 echo "License header check complete"
 '''
 
+[tasks.generate-readmes]
+description = "Automatically generate README.md for each component."
+category = "ICU4X Development"
+script_runner = "@duckscript"
+script = '''
+glob_pattern = set "./components/**/README.md"
```

Comment:
Nitpick: Include all crates, not just those under *components*.  We could get the full list by:

- Reading the list of packages from top-level Cargo.toml
- Searching not only `components` but also `utils`, `resources` (issue: could get stale)
- Is there some Cargo command for this?

Reply:
The presence of a Cargo.toml file in a subdirectory seems like it should work for this.